torch/csrc/Module.cpp

```diff
@@ -920,6 +920,29 @@ static PyObject* THPModule_setDeterministicAlgorithms(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPModule_setAllowTF32Onednn(
+    PyObject* _unsued,
+    PyObject* arg) {
+  HANDLE_TH_ERRORS
+  TORCH_CHECK(
+      PyBool_Check(arg),
+      "set_allow_tf32_onednn expects a bool, "
```

Comment:
```suggestion
      "_set_onednn_allow_tf32 expects a bool, "
```

Reply:
Thanks for you reminding, the code has been changed here